starters/prisma/wrangler.toml

```diff
@@ -0,0 +1,36 @@
+#:schema node_modules/wrangler/config-schema.json
+name = "__change_me__"
+main = "src/worker.tsx"
+compatibility_date = "2024-09-23"
+compatibility_flags = ["nodejs_compat"]
+assets = { binding = "ASSETS", directory = "public" }
+
+workers_dev = false
+routes = [
+  { pattern = "__change_me__", custom_domain = true }
+]
+
+[observability]
+enabled = true
+
+[[d1_databases]]
+binding = "DB"
+database_name = "__change_me__"
+database_id = "__change_me__"
+
+[[r2_buckets]]
+bucket_name = "__change_me__"
+binding = "R2"
+
+[durable_objects]
+bindings = [
+  { name = "SESSION_DO", class_name = "SessionDO" }
+]
```

Comment:
It looks like this starter doesn't include this file (in billable its [here](https://github.com/redwoodjs/reloaded/blob/03dd0ef88c9ed7a4154e5f321e3c65130bc268dc/experiments/billable/src/session.tsx), and [reexported in worker.tsx here](https://github.com/redwoodjs/reloaded/blob/498dec60e44b33b21e2b9c96f4dc52bb433f241e/experiments/billable/src/worker.tsx#L14)).

I don't think we necessarily need it for this starter though, maybe we can just remove this durable object configuration here?